Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -161,9 +167,18 @@ extension DiagnosticMessage where Self == StaticParserError {
   public static var missingColonInTernaryExpr: Self {
     .init("expected ':' after '? ...' in ternary expression")
   }
+  public static var missingConditionInIfStatement: Self {
+    .init("missing condition in 'if' statement")
+  }
+  public static var missingConditionInWhileStatement: Self {
+    .init("missing condition in 'while' statement")
+  }
   public static var missingConformanceRequirement: Self {
     .init("expected ':' or '==' to indicate a conformance or same-type requirement")
   }
+  public static var missingExpressionInSwitchStatement: Self {
+    .init("expected expression in 'switch' statement")
+  }
```

Comment:
Should we add a single error with this error message?

Reply:
Yes, I think so